modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/prepare/DdlPlan.java

```diff
@@ -45,7 +45,7 @@ public Type type() {
     /** {@inheritDoc} */
     @Override
     public ResultSetMetadata metadata() {
-        return List::of;
+        return ResultSetMetadataImpl.NO_METADATA;
```

Comment:
let's introduce a constant for DDL meta with a single column of type Bool and name "Applied"

Reply:
Got it, thanks.

Reply:
> let's introduce a constant for DDL meta.
Seems, we can do the same with DML, isn't it?

Reply:
Yes, indeed

Reply:
Well, as I could see, it's already derived from a query plan... So, it's up to you to change this to predefined meta or leave it as is